src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -1,19 +1,101 @@
 @import '~@pxblue/colors/palette.scss';
 
-.extended-fab-button {
-    width: 180px;
-    height: 56px;
-    padding: 0 26px;
-    border-radius: 29px;
-    transition-property: unset;
-    transition-duration: 1s;
-
-    .mat-icon {
-        margin-right: 8px;
+section.mat-typography {
+    padding: 16px 24px;
+    .example-section {
+        margin-bottom: 16px;
+    }
+    .show-text {
+        .text-container {
+            opacity: 1;
+            visibility: visible;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+        }
+        .spinner-container {
+            position: absolute;
+            opacity: 0;
+            visibility: hidden;
+            -webkit-transition: opacity 200ms, visibility 200ms;
+            transition: opacity 200ms, visibility 200ms;
+            margin-top: -28px;
```

Comment:
I'm not loving the positioning by way of absolute positioning and a bunch of negative margins. I feel like three should be a simpler way to get this layout right.

Reply:
Removed position: absolute css